.gitignore

```diff
@@ -1,5 +1,6 @@
 yarn.lock
 package-lock.json
+.DS_Store
```

Comment:
```suggestion
```
Rather than putting OS-specific exclusions into `.gitignore` (it looks like you're on a Mac) I'd recommend either not checking in this folder or [adding it to a global excludes file](https://stackoverflow.com/questions/7335420/global-git-ignore).